### PR TITLE
Update de_DE.js

### PR DESCRIPTION
**Pull request by Abstraktikus**

### CODE REVIEW

File: loc/de_DE.js [hunk 1]
```diff
@@ -226,24 +226,24 @@ module.exports = {
...
   lndViewInvoice: {
-    wasnt_paid_and_expired: 'This invoice was not paid for and has expired',
-    has_been_paid: 'This invoice has been paid for',
-    please_pay: 'Please pay',
+    wasnt_paid_and_expired: 'Diese Rechnung ist unbezahlt und abgelaufen.',
+    has_been_paid: 'Diese Rechnung wurde bezahlt.',
+    please_pay: 'Bitte zahle',
     sats: 'sats',
-    for: 'For:',
+    for: 'für:',
```
Comment (jstrnbrg) [inline loc/de_DE.js]: English is capitalised so I guess German should be as well? -> "Für:"

File: loc/de_DE.js [hunk 2]
```diff
@@ -25,8 +25,8 @@ module.exports = {
       empty_txs1: 'Deine Transaktionen erscheinen hier',
       empty_txs2: 'Noch keine Transaktionen',
       empty_txs1_lightning:
-        'Lightning wallet should be used for your daily transactions. Fees are unfairly cheap and speed is blazing fast.',
-      empty_txs2_lightning: '\nTo start using it tap on "manage funds" and topup your balance.',
+        'Verwende das Lightning Wallet für Deine täglichen Bezahlungen. Lightning Transaktionen sind konkurrenzlos günstig und verblüffend schnell.',
```
Comment (jstrnbrg) [inline loc/de_DE.js]: "Bezahlungen" should be "Zahlungen" I'd say

File: loc/de_DE.js [hunk 3]
```diff
@@ -226,24 +226,24 @@ module.exports = {
...
     sameWalletAsInvoiceError:
       'Du kannst nicht die Rechnung mit der Wallet begleichen, die du für die Erstellung dieser Rechnung verwendet hast.',
   },
   pleasebackup: {
-    title: 'Your wallet is created...',
+    title: 'Ihr Wallet wird erstellt...',
     text:
-      "Please take a moment to write down this mnemonic phrase on a piece of paper. It's your backup you can use to restore the wallet on other device.",
-    ok: 'OK, I wrote this down!',
+      "Nimm Dir Zeit die mnemonischen Wörter zur späteren Wiederherstellung des Wallets aufzuschreiben. Die Wörter sind dien einziges Backup im Fall eines Geräteverlustes.",
```
Comment (JakobHafke) [inline loc/de_DE.js]: des Wallet -> der Wallet
Dir -> dir
dien ->dein

File: loc/de_DE.js [hunk 4]
```diff
@@ -190,14 +190,14 @@ module.exports = {
...
+    electrum_settings: 'Electrum Einstellungen',
+    electrum_settings_explain: 'Leer lassen, um den Standard zu verwenden.',
     save: 'Speichern',
     about: 'Über',
     language: 'Sprache',
     currency: 'Währung',
-    advanced_options: 'Advanced Options',
-    enable_advanced_mode: 'Enable advanced mode',
+    advanced_options: 'Erweiterte Optionen',
+    enable_advanced_mode: 'Erweiterter Modus verwenden',
```
Comment (jstrnbrg) [inline loc/de_DE.js]: "Erweiterter Modus verwenden" -> "Erweiterten Modus verwenden"

**Comment by jstrnbrg** [comment]
Just saw @weixelbaumer comment above.  Added the points that I missed to my review above

File: loc/de_DE.js [hunk 5]
```diff
@@ -190,14 +190,14 @@ module.exports = {
       'Bitte installier Lndhub, um mit deiner eigenen LND Node zu verbinden' +
       ' und setz seine URL hier in den Einstellungen. Lass das Feld leer, um Standard- ' +
       'LndHub\n (lndhub.io) zu verwenden',
-    electrum_settings: 'Electrum Settings',
-    electrum_settings_explain: 'Set to blank to use default',
+    electrum_settings: 'Electrum Einstellungen',
+    electrum_settings_explain: 'Leer lassen, um den Standard zu verwenden.',
```
Comment (JakobHafke) [inline loc/de_DE.js]: Leer lassen, um Voreinstellung zu verwenden.

File: loc/de_DE.js [hunk 6]
```diff
@@ -25,8 +25,8 @@ module.exports = {
       empty_txs1: 'Deine Transaktionen erscheinen hier',
       empty_txs2: 'Noch keine Transaktionen',
       empty_txs1_lightning:
-        'Lightning wallet should be used for your daily transactions. Fees are unfairly cheap and speed is blazing fast.',
-      empty_txs2_lightning: '\nTo start using it tap on "manage funds" and topup your balance.',
+        'Verwende das Lightning Wallet für Deine täglichen Bezahlungen. Lightning Transaktionen sind konkurrenzlos günstig und verblüffend schnell.',
+      empty_txs2_lightning: '\nDrücke zum Starten «Beträge verwalten», um das Wallet zu laden.',
```
Comment (JakobHafke) [inline loc/de_DE.js]: Um zu starten und deine Wallet aufzuladen, drücke "Gelder verwalten".

Comment (JakobHafke) [reply loc/de_DE.js]: Better: "Guthaben verwalten"

**Comment by Overtorment** [comment]
Resolved conflict & merged. Thanks!

**Pull request merged**